substrate/frame/support/test/tests/runtime_ui/pass/basic.rs

```diff
@@ -0,0 +1,37 @@
+// This file is part of Substrate.
```

Comment:
I thought we didn't need UI tests for happy paths?

Reply:
Yeah, this is just a sanity one just to follow the previous pattern.